src/screens/Main.tsx

```diff
@@ -58,7 +74,7 @@ export function Main({ className }: Props) {
         </Grid>
         <Grid container>
           <Grid item md={12}>
-            <Accounts />
+            {proj}
```

Comment:
this would replace can be replaced as `<NewComponent />`

Reply:
Agreed - will do that in later PR when Figma is prepared and we start building the actual compoentns